site/pages/disruption-detail/[disruptionId].page.tsx

```diff
@@ -370,19 +370,19 @@ const DisruptionDetail = ({
             ) : null}
             {duplicateDisruptionPopUpState && csrfToken ? (
                 <Popup
-                    action={`/api/duplicate-disruption${queryParams["template"] ? "?template=true" : ""}`}
+                    action={`/api/duplicate-disruption${disruption.template ? "?template=true" : ""}`}
                     cancelActionHandler={cancelActionHandlerDuplicateDisruption}
                     csrfToken={csrfToken}
                     hiddenInputs={duplicateDisruptionPopUpState.hiddenInputs}
                     continueText="Yes, duplicate"
                     cancelText="No, return"
                     questionText={`Are you sure you wish to duplicate the ${
-                        queryParams["template"] ? "template" : "disruption"
+                        disruption.template ? "template" : "disruption"
```

Comment:
currently we dont have a functionality to duplicate template

Reply:
removing it